inst/examples/e2e-test/app.R

```diff
@@ -1,88 +1,20 @@
 library(shiny)
 library(shiny.fluent)
 
+source("utils.R")
+source("toggle_visibility.R")
+source("scenarios.R")
+
 ui <- fluidPage(
-  h1("Inputs tests"),
-  actionButton("updateInputs", "Update"),
-  actionButton("toggle", "Toggle visibility"),
-  uiOutput("main"),
-  h1("UI Components tests"),
-  Stack(
-    div("placeholder")
-  ),
-  h1("Scenarios tests"),
-  div(
-    Pivot(
-      PivotItem(headerText = "Tab 1", Label("Hello 1")),
-      PivotItem(headerText = "Tab 2", Label("Hello 2")),
-      onLinkClick = JS("function() { alert('Pivot tab switched') }")
-    ),
-    Pivot(
-      PivotItem(headerText = "Tab 1", Label("Hello 1")),
-      PivotItem(headerText = "Initially hidden components",
-        div(
-          textInput("scenarioInput1", "Label"),
-          textOutput("scenarioTextOutput1")
-        )
-      )
-    )
-  ),
-  h2("Update button inside React"),
-  Stack(
-    div(
-      actionButton("updateInputs2", "Update text"),
```

Comment:
Ditto - please add `actionButton` to the "Shiny in React" section to test if it works.